tests/wpt/include-layout-2020.ini

```diff
@@ -19,5 +19,7 @@ skip: true
       skip: false
   [css-backgrounds]
     skip: false
+  [filter-effects]
+    skip: false
```

Comment:
Also add `css-color` for the `opacity` property?

Reply:
Okay. Done.